gravatar-ui/src/main/java/com/gravatar/ui/components/MiniProfileCard.kt

```diff
@@ -0,0 +1,75 @@
+package com.gravatar.ui.components
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ProvideTextStyle
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.gravatar.api.models.UserProfile
+import com.gravatar.ui.components.atomic.Avatar
+import com.gravatar.ui.components.atomic.DisplayName
+import com.gravatar.ui.components.atomic.Location
+import com.gravatar.ui.components.atomic.ViewProfileButton
+
+@Composable
+fun MiniProfileCard(profile: UserProfile, modifier: Modifier = Modifier) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.Start,
+        verticalArrangement = Arrangement.Top,
+    ) {
+        Row {
+            Avatar(
+                profile = profile,
+                size = 72.dp,
+                modifier = Modifier.clip(CircleShape),
+            )
+            Column(modifier = Modifier.padding(14.dp, 0.dp, 0.dp, 0.dp)) {
```

Comment:
⛏
```suggestion
            Column(modifier = Modifier.padding(start = 14.dp)) {
```

Should we also add the padding at the end?

```suggestion
            Column(modifier = Modifier.padding(horizontal = 14.dp)) {
```

Reply:
> Should we also add the padding at the end?

I'd keep the start padding only, it seems easier to manage in a card like in there:

<img width="383" alt="Screenshot 2024-04-08 at 14 56 53" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/40213/5ed5eb54-5437-4049-876d-fbc2d3accd24">



(24dp padding start and end on the card)